src/Impostor.Api/Net/Messages/S2C/Message24EndGameHostMigrationS2C.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace Impostor.Api.Net.Messages.S2C
+{
+    public class Message24EndGameHostMigrationS2C
+    {
+        public static void Serialize(IMessageWriter writer, int gameCode, int hostClientId)
+        {
+            writer.StartMessage(MessageFlags.EndGameHostMigration);
+            writer.Write(gameCode);
+            writer.Write(hostClientId);
+            writer.EndMessage();
+        }
+
+        public static void Deserialize(IMessageReader reader)
+        {
+            throw new NotImplementedException();
+        }
+    }
+}
```

Comment:
```suggestion
using Impostor.Api.Games;

namespace Impostor.Api.Net.Messages.S2C
{
    public class Message24EndGameHostMigrationS2C
    {
        public static void Serialize(IMessageWriter writer, GameCode gameCode, int hostClientId)
        {
            writer.StartMessage(MessageFlags.EndGameHostMigration);
            gameCode.Serialize(writer);
            writer.Write(hostClientId);
            writer.EndMessage();
        }

        public static void Deserialize(IMessageReader reader, out GameCode gameCode, out int hostClientId)
        {
            gameCode = reader.ReadInt32();
            hostClientId = reader.ReadInt32();
        }
    }
}
```

Reply:
What a terrible diff, did you just change the type of gameCode?

Reply:
That and I also implemented `Deserialize`, because might as well.

Reply:
Applied suggestion